### PR TITLE
Fix EZP-24441: Implement FieldDefinition form mapper for ezbinaryfile

### PR DESCRIPTION
**Pull request by glye**
Manual tests.

Depends on changes in Type and Converter, see: https://github.com/ezsystems/ezpublish-kernel/pull/1283

https://jira.ez.no/browse/EZP-24441

### CODE REVIEW

**Comment by andrerom** [comment]
+1

File: lib/FieldType/Mapper/BinaryFileFormMapper.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        $fieldDefinitionForm
+            ->add(
+                'maxSize',
+                'integer',
+                [
+                    'required' => true,
+                    'property_path' => 'validatorConfiguration[FileSizeValidator][maxFileSize]',
+                    'label' => 'field_definition.ezbinaryfile.max_file_size',
+                ]
+            );
```
Comment (lolautruche) [inline lib/FieldType/Mapper/BinaryFileFormMapper.php]: CS. PSR-2 suggests the following:

```php
$fieldDefinitionForm->add('maxSize', 'integer', [
    'required' => true,
    'property_path' => 'validatorConfiguration[FileSizeValidator][maxFileSize]',
    'label' => 'field_definition.ezbinaryfile.max_file_size',
]);
```

You gain one indent :wink:

**Comment by lolautruche** [comment]
Except CS remark, +1

File: lib/FieldType/Mapper/BinaryFileFormMapper.php (hunk 1 above)
Comment (glye) [inline lib/FieldType/Mapper/BinaryFileFormMapper.php]: @lolautruche Interesting. PhpStorm says "Only one argument is allowed per line in a multi-line function call", though not if only the array declaration is multi-line. Ok, I'll do this.

**Comment by glye** [comment]
CS remark fixed.

**Pull request merged**